.husky/pre-push

```diff
@@ -1,6 +1,7 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
+npx npx sort-package-json
```

Comment:
Seems there is an additional `npx`. Should it be required?

Reply:
Good catch .... just fixed it